.travis.yml

```diff
@@ -1,2 +1,3 @@
 language: r
 cache: packages
+NOT_CRAN: true
```

Comment:
So instead of `NOT_CRAN` it's more straightforward to check for existence or non-existence of the `SIGOPT_API_TOKEN` environment variable. Although, this makes me wonder how many times users will run the test package with their real API token set and accidentally create experiments.

I think we can use this if we change the name from `NOT_CRAN` to something like `RUN_API_TEST` and comment that for that variable to be true the `SIGOPT_API_TOKEN` must be set and that we prefer development tokens for test purposes than this should be ok.